lib/resque/durable.rb

```diff
@@ -46,9 +55,17 @@ def heartbeat(args)
 
     def around_perform_manage_audit(*args)
       if a = audit(args)
-        a.heartbeat!
         return if a.complete?
-        yield
+        if background_heartbeat_interval
+          raise "background_heartbeat_interval (#{background_heartbeat_interval.inspect}) be smaller than job_timeout (#{job_timeout.inspect})" if background_heartbeat_interval >= job_timeout
```

Comment:
Can this be done on the job_timeout / background interval accessors? I feel like it'd be safer on startup, but these are durable jobs so I guess it doesn't matter.